src/jdk.incubator.vector/share/classes/jdk/incubator/vector/VectorSpecies.java

```diff
@@ -154,6 +154,31 @@
      */
     int loopBound(int length);
 
+    /**
+     * Loop control function which returns the largest multiple of
+     * {@code VLENGTH} that is less than or equal to the given
+     * {@code length} value.
+     * Here, {@code VLENGTH} is the result of {@code this.length()},
+     * and {@code length} is interpreted as a number of lanes.
+     * The resulting value {@code R} satisfies this inequality:
+     * <pre>{@code R <= length < R+VLENGTH}
+     * </pre>
+     * <p> Specifically, this method computes
+     * {@code length - floorMod(length, VLENGTH)}, where
+     * {@link Math#floorMod(long,int) floorMod} computes a remainder
+     * value by rounding its quotient toward negative infinity.
+     * As long as {@code VLENGTH} is a power of two, then the result
+     * is also equal to {@code length & ~(VLENGTH - 1)}.
+     *
+     * @param length the input length
+     * @return the largest multiple of the vector length not greater
+     *         than the given length
+     * @throws IllegalArgumentException if the {@code length} is
+    negative and the result would overflow to a positive value
```

Comment:
Just spotted this, small misprint here, no `*` in comment.

Reply:
Thanks. Fixed.